MigrateUtil.go

```diff
@@ -21,6 +23,9 @@ type MigrateConfig struct {
 	Host          string `env:"DB_HOST"  envDefault:"localhost"`
 	Port          string `env:"DB_PORT"  envDefault:"5432"`
 	DbName        string `env:"DB_NAME"  envDefault:"migrate_test"`
+	EnableCounter string `env:"ENABLE_VALIDATE_COUNTER" envDefault:"true"`
+	RetryDelay    string `env:"RETRY_DELAY" envDefault:"5"`
```

Comment:
rename retry parameters to some more apt name. Maybe can include DB_CONNECTION_ to it. Also update default values as per ideal case.